src/MICore/Debugger.cs

```diff
@@ -569,6 +571,13 @@ public Task CmdBreakInternal()
                     return CmdUnixBreak(debuggeePid, ResultClass.done);
                 }
             }
+            else if (_transport is PipeTransport)
+            {
+                if (((PipeTransport)_transport).Interrupt(PidByInferior("i1")))
```

Comment:
Probably fine to throw since it would be a very bad situation when thread group 'i1' wasn't found as that is the debuggee processes thread group. However I'll add the check.